bitnami/discourse/values.yaml

```diff
@@ -0,0 +1,276 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
```

Comment:
After checking the values, I think it may make more sense to divide it into two big sections:

`discourse`

`sidekiq`

In each of the sections you can define the replicas, the extra envs, the sidecars, the initcontainers, the resources, the liveness probes... etc.

If you think it's a big change I can do it myself.